src/rdkafka_topic.h

```diff
@@ -69,7 +69,7 @@ struct rd_kafka_itopic_s {
 	} rkt_state;
 
         int               rkt_flags;
-#define RD_KAFKA_TOPIC_F_LEADER_UNAVAIL   0x1 /* Leader lost/unavailable
+#define RD_KAFKA_TOPIC_F_LEADER_UNAVAIL   0x1 /* Broker lost/unavailable
```

Comment:
This still depicts the actual leader, not a replica, right? (so the comment should not be changed)

Reply:
yep, that's a mistake.